### ISSUE TITLE
Can you share the pretrain process logs?

### ISSUE THREAD

**Issue by Dragonzz27** [VisualComputingInstitute/DITR#3]
Thanks for your great work.
I also work on cross modality learning (Image and Point Cloud), but on pretraining part only.
I wonder whether to keep feature distribution from Image and PointCloud similar is helpful to pretraining process.
What's more, how many epochs should be done to make this work.
So, i would be grateful if you can share the Pretrain Logs.

**Comment by kabouzeid** [comment]
Which dataset setting would you be most interested in? I will look for the respective log file

**Comment by Dragonzz27** [comment]
> Which dataset setting would you be most interested in? I will look for the respective log file

Only ScanNet will be fine，thank you！ʕ•ᴥ•ʔ

**Comment by YilmazKadir** [comment]
- **Joint Pretraining on ScanNet + Structured3D**  
  We align point features with DINOv2 features using a cosine similarity loss.  
  [📄 Pretraining Log](https://drive.google.com/file/d/1aiDgIg8A0RBBd-LQlbT9t9UBvJ9ejqXr/view?usp=sharing)

- **Fine-tuning on ScanNet**  
  We fine-tune the above pre-trained model on the ScanNet semantic segmentation task.
  [📄 Fine-tuning Log](https://drive.google.com/file/d/16fgv8uJcbME5gaOL5QxygLid0N0vGecW/view?usp=sharing)

Feel free to reach out if you have any further questions.

**Comment by Dragonzz27** [comment]
Thank you very much for sharing the training logs. We find your work truly inspiring.
We have already demonstrated the effectiveness of the pre-training method on the SparseUNet model.

![Image](https://github.com/user-attachments/assets/5ed3648c-3bb2-489b-88ee-3d6aa3baba02)
Red: SparseUNet training from scratch
Orange: SparseUNet training using MaskedSceneContrast pretrain weight
Blue: SparseUNet training using DITR pretrain weight

We will cite your paper in our  future work and hope your paper gets accepted soon.
